### PR TITLE
misc: improve the readability of misc/git-tag-maybe.sh

### PR DESCRIPTION
**Pull request by leleliu008**

### CODE REVIEW

File: misc/git-tag-maybe.sh [hunk 1]
```diff
@@ -1,36 +1,46 @@
...
+}
+
+run() {
+    printf '%b\n' "${COLOR_PURPLE}==>${COLOR_OFF} ${COLOR_GREEN}$*${COLOR_OFF}"
+    eval "$*"
+}
+
+run git --version
+
 base=5.9
```
Comment (masatake) [inline misc/git-tag-maybe.sh:21]: Could you move the definition of `${base}` to the beginning of the script?

**Review by masatake — APPROVED** [review]

**Comment by masatake** [comment]
Let's see what happens this weekend!

**Comment by leleliu008** [comment]
It will be like this:

Screenshot from 2022-05-26 17-58-51

**Comment by masatake** [comment]
Fine!

**Pull request merged**